src/config/reviews.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Review provider Configurations
+    |--------------------------------------------------------------------------
+    */
+
+    'reviewsCoUk' => [
```

Comment:
this array indentation is different than the one used on other files

Reply:
It's not Immediately clear to me the difference I had a look in installs tools but couldn't see anything that stood out to me